esphome/components/seeed_mr24hpc1/binary_sensor.py

```diff
@@ -0,0 +1,24 @@
+import esphome.codegen as cg
+from esphome.components import binary_sensor
+import esphome.config_validation as cv
+from esphome.const import (
+    DEVICE_CLASS_OCCUPANCY,  # https://github.com/limengdu/esphome/blob/17e1d4c2455997a0a15c875950be1942c6249d0d/esphome/const.py#L1006
+)
+from . import CONF_MR24HPC1_ID, MR24HPC1Component
+
+AUTO_LOAD = ["seeed_mr24hpc1"]
+CONF_SOMEONE_EXIST = "someone_exist"
```

Comment:
As per a suggestion I saw from @kbx81 it is a bit better to reuse config keys from other components to not create so much diversity/confusion for users
```suggestion
CONF_HAS_TARGET = "has_target"
```